Design rules/Policy Design Rules.md

```diff
@@ -0,0 +1,68 @@
+# Policy Design Rules
+
+In dit document beschrijven we hoe we in het Kenniscentrum Standaarden van VNG Realisatie omgaan met Design Rules. 
+Daarbij doelen we op de eigen Design Rules, de Design Rules die in de Nederlandse API Strategie zijn beschreven maar ook op de Extensions die op beide niveaus beschikbaar zijn.
+Extensions hebben geen officiële status, ze kunnen worden gezien als potentiële Design Rules.
+
+## Toepassing Design Rules
+
+Het toepassen van Design Rules en Extensions is aan de volgende regels gebonden:
+
+1. Bij het vervaardigen van API specificaties houden we ons aan [de Design Rules zoals gedefinieerd in de Nederlandse API Strategie](https://docs.geostandaarden.nl/api/API-Designrules/);
+2. Tevens houden we ons aan [de Design Rules van VNG Realisatie](https://github.com/VNG-Realisatie/API-Kennisbank/blob/master/Design%20rules/readme.md);
```

Comment:
Wat is de verhouding tussen de Nederlandse API Strategie en de VNG Realisatie Design Rules?  Wanneer een design rule afwijkt van de API strategie, telt volgen van de VNG R Design Rules dan als "leg uit" om af te wijken van de Nederlandse API Strategie?

Reply:
Goede observatie. De vraag die we daarvoor als eerste moeten beantwoorden is of we het acceptabel vinden dat er Design Rules zijn van VNG Realisatie die afwijken van Design Rules in de Nederlandse API Strategie.

Reply:
In mijn beleving moeten wij een heel erg goed verhaal hebben om structureel af te wijken van de landelijke Rest-API-strategie. Zeldzaam geval dus. 

Mocht het voorkomen dan is dat inderdaad ook een "Leg uit"-situatie. In dat geval moeten we ook weer de discussie met de rest van de community opstarten om te kijken waar deze verschillen in inzicht in zitten en of we dan de landelijke strategie dan niet iets losser moeten definiëren.

Reply:
Ik heb bullet 12 in de sectie 'Het toevoegen, wijzigen of verwijderen van VNG Realisatie Design Rules en Extensions' toegevoegd.